Dockerfile

```diff
@@ -1,20 +0,0 @@
-FROM cern/cc7-base
```

Comment:
This will delete Docker file from main. Just copy Dockerfile content from main and add the new packages to required_packages.txt. There should be no need to copy over any more folders if you followed the correct folder structure. 

Please also add to line `run mkdir utils scripts configs` to become `run mkdir utils scripts configs misc` since this is a bug in main